src/System.Collections.NonGeneric/ref/System.Collections.NonGeneric.cs

```diff
@@ -100,12 +99,12 @@ public partial class Queue : System.Collections.ICollection, System.Collections.
         public virtual object Clone() { throw null; }
         public virtual bool Contains(object obj) { throw null; }
         public virtual void CopyTo(System.Array array, int index) { }
-        public virtual object Dequeue() { throw null; }
-        public virtual void Enqueue(object obj) { }
+        public virtual object? Dequeue() { throw null; }
```

Comment:
should Conatins take nullable?

Reply:
No as that parameter is key which shouldn't be nullable